sal_interfaces/ATPtg/ATPtg_Telemetry.xml

```diff
@@ -506,7 +485,7 @@
     </item>
     <item>
         <EFDB_Name>sunsetTime</EFDB_Name>
-        <Description>Sunset time (UTC)</Description>
+        <Description>Sunset time (LOCAL)</Description>
```

Comment:
Please don't change to local time. Design where you have absolute, not changing UTCs fares much better then designs with time changes. You can easily convert UTC to localtime for displays. It's hard to change local time to UTC, particularly if you don't know what's timezone it was produced for.

Reply:
OK, we will change it back to UTC.
Currently, sunsetTime is type ‘string’ . Should we change it to absolute timestamp (TAI seconds)? If so, we will remove the timeToSunset.

Reply:
I think TAI in seconds would be best.

Reply:
Agree, TAI in seconds. LBTO has string fields for various display-only stuff (e.g there is AZ radian degrees and string for DDD:MM:SS.ss display). I would rather stick with numeric values only - GUIs have enought CPU power to translate those (just bear in mind rounding problems, eg. don't display 10:60:00.00 for 10.9999999999 degrees, it really needs to be 11:00:00.00 - properly rounding and watching for overflows is the key).